core/src/main/java/com/nuodb/migrator/cli/validation/NuoDBConnectionGroupValidator.java

```diff
@@ -82,11 +82,7 @@ protected void validateUsername(CommandLine commandLine, Option option, String u
 
     @Override
     protected void validatePassword(CommandLine commandLine, Option option, String password) {
-        if (isEmpty(password)) {
-            throw new OptionException(
-                    format("Missing required option %s. The user's password should be provided", getPasswordOption()),
-                    option);
-        }
+        // Validated in PasswordOptionProcessor
```

Comment:
Is there any harm in validating the password twice? Do we have to perform validation in the PasswordOptionProcessor?

Reply:
Doing it here is technically possible but not very nice because this is a validator for an entire group of options not a single password option. I preferred to use a separate class to extract a password variable from the password option.